test/emitter.test.ts

```diff
@@ -0,0 +1,88 @@
+// Note: we need to import logger from the root
+// because this is the logger used in logError in ../src/common/util
+import { logger } from "../node_modules/@coder/logger"
+
+import { Emitter } from "../src/common/emitter"
+
+describe("emitter", () => {
+  describe("Emitter", () => {
```

Comment:
Why double `describe`?

Reply:
eh i was trying to follow a pattern used in other places 😂 i'll clean this up